metrics-dropwizard/src/main/scala/com/mesosphere/usi/metrics/dropwizard/reporters/StatsDReporter.scala

```diff
@@ -69,7 +81,7 @@ class StatsDReporter(settings: StatsdReporterSettings, registry: MetricRegistry)
   private def reportGauge(socket: ActorRef, name: String, gauge: Gauge[_]): Unit = {
     val value: Number = gauge.getValue match {
       case v: Double => if (v.isNaN) 0.0 else v
-      case v: Float => if (v.isNaN) 0.0 else v.toDouble
+      case v: Float  => if (v.isNaN) 0.0 else v.toDouble
```

Comment:
This could be a conflict generating machine? I'd turn alignment off unless if this alignment only happens for one-liners.